web/html/media/video-formats.md

```diff
@@ -52,15 +52,12 @@ AddType video/webm .webm
 
 Finish the sentence:
 
-In order for videos today to be supports on all major web browsers you should use either the `???` or `???` format.
-
-* .mp4
-* .webm
-* h.246
-* codec
-* .mp3
-* .oog
-* flash
+In order for videos today to be supported on all major web browsers you should use either the `???` format.
+
+* .mp4 or.webm
```

Comment:
```suggestion
* .mp4 or .webm
```

Reply:
It took my a while to see there is no space there😅